tests/test_http.py

```diff
@@ -19,7 +19,7 @@
 Unit tests for the googleapiclient.http.
 """
 from __future__ import absolute_import
-from six.moves import range
+from six.moves import range, xrange
```

Comment:
Oops, I missed that the test module was already importing `range`.  You can back out that second commit.

Reply:
Done